### ISSUE TITLE
migrating to 6.0.0-preview.3.21201.2 from 6.0.0-preview.2.21154.2 break asp core mvc template

### ISSUE THREAD

**Issue by tschmit** [dotnet/aspnetcore#32675]
### Describe the bug
web app does not start after upgrading the packages.

The error appear on the following line in startup.cs:

    services.AddDefaultIdentity<IdentityUser>(options => options.SignIn.RequireConfirmedAccount = true)
                .AddEntityFrameworkStores<ApplicationDbContext>();

### To Reproduce
Start a new project c# asp net core mvc
Update all nuget pakage
Run with IIS Express

### Exceptions (if any)
System.TypeLoadException
  HResult=0x80131522
  Message=Could not load type 'Microsoft.AspNetCore.Mvc.ApplicationParts.ConsolidatedAssemblyApplicationPartFactory' from assembly 'Microsoft.AspNetCore.Mvc.Razor'.
  Source=<Impossible d'évaluer la source de l'exception>
  Arborescence des appels de procédure :
<Impossible d'évaluer l'arborescence des appels de procédure de l'exception>

### Further technical details
you can mitigate the error by downgrading to previous version:
- Microsoft.AspNetCore.Identity.EntityFrameworkCore, and
- Microsoft.AspNetCore.Identity.UI

**Comment by mkArtakMSFT** [comment]
Translated the exception using Bing Translator:
```
Source-<Impossible to assess the source of the exception>
Case of procedural appeals:
<Impossible to assess the tree of exception procedural appeals>
```

**Comment by mkArtakMSFT** [comment]
Thanks for contacting us.
Can you please share the upgraded template project? Thanks!

**Comment by tschmit** [comment]
I haven't updated the template, I downgrad the packages to stay compatible with the template.

**Comment by tschmit** [comment]
it is quite blocking as the package update is trigerred by Identity Scaffolding. I had to downgrad the package after the scaffold.

**Comment by tschmit** [comment]
problem solved after updating sdk from 6.0.100-preview.2.21155.3 to 6.0.100-preview.3.21202.5
